### PR TITLE
[Create a subscription] - Tab Meeting Recording and Transcript (Csharp)

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline  - 

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6747817813

image

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline -**

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6767466754

image

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline -
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6861030512

image

File: samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/ClientApp/src/components/meetingtranscriptrecording.jsx [hunk 1]
```diff
@@ -26,7 +31,35 @@ const MeetingTranscriptRecording = () => {
...
+    useEffect(() => {
+        const newConnection = new HubConnectionBuilder()
+            .withUrl(`${window.location.origin}/chatHub`)
+            .withAutomaticReconnect()
+            .build();
+
+        setConnection(newConnection);
+    }, []);
+
+
```
Comment (Pawank-MSFT) [inline samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/ClientApp/src/components/meetingtranscriptrecording.jsx:46]: remove extra spaces, check other places also.

File: samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/Images/Policy.png
Comment (Pawank-MSFT) [inline samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/Images/Policy.png:1]: Hide AppIds, Password, ObjectIds, TenantId etc.

**Review by Pawank-MSFT — COMMENTED** [review]
Gave some small comments.

File: samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/ClientApp/src/components/meetingtranscriptrecording.jsx (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/ClientApp/src/components/meetingtranscriptrecording.jsx]: Fixed

File: samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/Images/Policy.png
Comment (Harikrishnan-MSFT) [reply samples/tab-meeting-transcript-recording/csharp/MeetingTranscriptRecording/Images/Policy.png:1]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]
approved

**Comment by Pawank-MSFT** [comment]
Executed build pipeline
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/7026925448

**Comment by Harikrishnan-MSFT** [comment]
**Executed build pipeline -** 

image

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**